src/main/java/Main.java

```diff
@@ -1,9 +1,34 @@
 /**
  * Created by iyasuwatts on 10/17/17.
  */
+import java.util.*;
 public class Main {
 
     public static void main(String[] args){
+        int result=0;
+        Scanner input = new Scanner(System.in);
+        System.out.println(" please enter a number and I willl give you the sum or product of numbers between 1 and the " +
+                "number you put");
+        int inputNumber = input.nextInt();
+        input.nextLine();
+        System.out.println("would you like a sum or a product?");
+        String inputAnswer = input.nextLine();
+        if(inputAnswer.equalsIgnoreCase("Sum")){
+            for(int i = 1;i<=inputNumber;i++){
+                result+=i;
+            }
+        }else{
+            result+=1;
```

Comment:
I think you should have checked the input for Product here. Also you should rework to ask sum or product before asking for the number.